testing/test_terminal.py

```diff
@@ -97,6 +97,20 @@ def test_func():
             ["    def test_func():", ">       assert 0", "E       assert 0"]
         )
 
+    def test_parametrize_relative_path(self, testdir, option):
+        testdir.makepyfile(
+            """
+            import pytest
+
+            @pytest.mark.parametrize("param", [("/p/../../", )])
+            def test_func(param):
+                pass
+            """
+        )
+        testdir.runpytest(*Option(verbosity=1).args).stdout.fnmatch_lines(
+            ["*test_parametrize_relative_path.py::test_func[/p/../../] PASSED*"]
```

Comment:
actually not sure what this should be - perhaps 
```suggestion
            ["*test_parametrize_relative_path.py::test_func[p] PASSED*"]
```

Reply:
not sure what cleanup is *intended*